### ISSUE TITLE
Voiceover is not announcing Number of suggestions were found while searching any key in "keyboard shortcut edit field": A11y_Visual Studio Code Client_Customizations and settings_Keyboard Shortcuts_Screen Reader. 

### ISSUE THREAD

**Issue by keerthiduvvuri** [microsoft/vscode#221565]
### GIT Hub Tags:
#A11yWCAG2.1; #A11yTCS; #A11ySev4; #Visual Studio Code Client; #BM_Visual Studio Code Client_MAC_JULY2024; #DesktopApp; #WCAG4.1.3; #FTP; #A11yMAS; #Status Message; #MAC; #Device; #Voiceover; #A11yUsable;

### Environment and OS details:
MAC Book Pro
MAC OS Sonoma Version 14.5
Visual Studio code Insider version 1.92.0

### Reproduction Steps:

1. Open Visual Studio Code Insiders editor
2. Tab Navigate to setting icon.
3. Select "Keyboard shortcuts options" and press enter or Press keyboard shortcut as command K, Command S
4. Keyboard shortcut table gets opened.
5. Tab till "Keyboard Shortcut Edit field".
6. Type any Text or Key and Observe that Voiceover is not announcing Number of suggestions were found while searching any key in "keyboard shortcut edit field".

### Actual behavior:
Voiceover is not announcing Number of suggestions were found while searching any key in "keyboard shortcut edit field".

### Expected behavior:
Voiceover should be announcing Number of suggestions were found while searching any key in "keyboard shortcut edit field".

### User Impact:
Screen reader users will face difficulty if Voiceover is not announcing Number of suggestions were found while searching any key in "keyboard shortcut edit field".

### Attachments:

https://github.com/user-attachments/assets/00b8d197-5447-4c44-ab81-f814ad889caa

**Comment by meganrogge** [comment]
@sandy081 do you know if we have ever had support for this?

**Comment by PoojaNamde** [comment]
#Rev:poojanamde;

**Comment by keerthiduvvuri** [comment]
Decreasing to Sev4 according to Review of Indu Priya Setty.
Thanks!!
